### ISSUE TITLE
0.16.0 gradle-avro-plugin is not imported

### ISSUE THREAD

**Issue by changgunyee** [davidmc24/gradle-avro-plugin#155]
**Prerequisites**

* [ ] Are you running the latest version of the plugin? (Check [releases](https://github.com/davidmc24/gradle-avro-plugin/releases))
* [x] Are you running a supported version of Gradle? (Check the [README](https://github.com/davidmc24/gradle-avro-plugin/blob/master/README.md))
* [x] Are you running a supported version of Apache Avro? (Check the [README](https://github.com/davidmc24/gradle-avro-plugin/blob/master/README.md))
* [x] Are you running a supported version of Java? (Check the [README](https://github.com/davidmc24/gradle-avro-plugin/blob/master/README.md))
* [x] Did you check to see if an [issue](https://github.com/davidmc24/gradle-avro-plugin/issues) has already been submitted?
* [x] Are you reporting to the correct repository?  If your schema doesn't work with the Apache Avro CLI tool either, it's not a problem with this plugin.  Running your file through the `CLIComparisonTest` in the sample project under the `test-project` directory can help diagnose this.
* [x] Did you perform a cursory search?

For more information, see the [CONTRIBUTING](https://github.com/davidmc24/gradle-avro-plugin/blob/master/CONTRIBUTING.md) guide.

**Describe the bug**

We have been using gradle avro plugin well but today in jenkins build, avro plugin not imported no more.

**To Reproduce**
build.gradle
```kt
plugins {
    `java-library`
    `maven-publish`
    id("io.franzbecker.gradle-lombok") version "3.0.0"
    id("com.commercehub.gradle.plugin.avro") version "0.16.0"
//    kotlin("jvm") version "1.3.61"
}
````
gradle.properties
```kt
pluginManagement {
    repositories {
        gradlePluginPortal()
        jcenter()
        maven {
            name = "JCenter Gradle Plugins"
            url  = uri("https://dl.bintray.com/gradle/gradle-plugins")
        }
    }
}
```
error log
```
* What went wrong:
Plugin [id: 'com.github.davidmc24.gradle.plugin.avro', version: '0.16.0'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'com.github.davidmc24.gradle.plugin.avro:com.github.davidmc24.gradle.plugin.avro.gradle.plugin:0.16.0')
  Searched in the following repositories:
    Gradle Central Plugin Repository
    BintrayJCenter
    MavenRepo
    JCenter Gradle Plugins(https://dl.bintray.com/gradle/gradle-plugins)
    maven(http://nexus.coupang.net/content/groups/public/)
```

1. run build
2. error found

Please provide complete input files that reproduce the problem, not fragments.
When possible, please express this using `test-project`.

**Expected behavior**

build well

**Environment (please complete the following information):**
 - Gradle Version [e.g. 5.6.1] 6.0.1
 - Apache Avro Version [e.g. 1.8.2] 1.8.2
 - Gradle-Avro Plugin Version [e.g. 0.17.0] 0.16.0
 - Java Version [e.g. 13.0.2] Java8
 - OS: [e.g. Mac OS X Mojave, Windows 10, Ubuntu 16.04 ] Ubuntu 16.04

**Additional context**

**Comment by davidmc24** [comment]
Yes, this is almost certainly related to the JCenter sunset. I have no control over how they manage JCenter. The options I can offer are either you can download the artifacts for the version you're currently using from GitHub Releases and host them for yourself privately, or you can update to version 1.x of the plugin and resolve it from Maven Central.
